contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.4.0--3.5.0.sql

```diff
@@ -167,6 +167,412 @@ EXCEPTION WHEN OTHERS THEN
 END;
 $$;
 
+create or replace view sys.all_objects as
```

Comment:
Non-blocking: I think we can ignore 3.4.0--3.5.0 upgrade script for now since it should actually be removed from PG16.1

Reply:
So, we don't need to have any 3.x --> 3.y upgrade script in 4_X_DEV and 4_0_STABLE branches? If so, I can remove this file in this PR.